tests/pyfunc/test_virtualenv.py

```diff
@@ -134,6 +134,27 @@ def test_differenet_requirements_create_different_environments(temp_mlflow_env_r
     assert len(list(temp_mlflow_env_root.iterdir())) == 2
 
 
+@use_temp_mlflow_env_root
+def test_environment_directory_is_cleaned_up_when_unexpected_error_occurs(
+    temp_mlflow_env_root, sklearn_model
+):
+    sklearn_req = "scikit-learn==999.999.999"
```

Comment:
83 years from now someone is going to wonder why this test fails ;)

Reply:
I don't think that will happen. You normally don't release 999 minor/micro versions :)